Resources/Prototypes/_SpaceCats/name_identifier_groups.yml

```
@@ -0,0 +1,4 @@
+# used by the nanochatcard numbers
+- type: nameIdentifierGroup
+  id: NanoChat
+  maxValue: 9999
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)